src/libraries/Microsoft.PowerFx.Connectors/ConfigExtensions.cs

```diff
@@ -15,50 +14,34 @@ namespace Microsoft.PowerFx
     [ThreadSafeImmutable]
     public static class ConfigExtensions
     {
-        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument)
-        {
-            return config.AddService(functionNamespace, openApiDocument, null, new ConnectorSettings());
-        }
-        
-        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument, HttpMessageInvoker httpClient)
-        {
-            return config.AddService(functionNamespace, openApiDocument, httpClient, new ConnectorSettings());
-        }        
-
-        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument, HttpMessageInvoker httpClient, ICachingHttpClient cache, bool numberIsFloat)
-        {
-            return config.AddService(functionNamespace, openApiDocument, httpClient, new ConnectorSettings() { Cache = cache, NumberIsFloat = numberIsFloat });
-        }
-
         /// <summary>
         /// Add functions for each operation in the <see cref="OpenApiDocument"/>. 
         /// Functions names will be 'functionNamespace.operationName'.
         /// Functions are invoked via rest via the httpClient. The client must handle authentication. 
         /// </summary>
+        /// 
         /// <param name="config">Config to add the functions to.</param>
-        /// <param name="functionNamespace">Namespace to place functions in.</param>
+        /// <param name="connectorSettings">Connector settings containing Namespace, NumberIsFloat and MaxRows to be returned.</param>        
         /// <param name="openApiDocument">An API document. This can represent multiple formats, including Swagger 2.0 and OpenAPI 3.0.</param>
-        /// <param name="httpClient">Required iff we want to invoke the API. A client to invoke the endpoints described in the api document. This must handle auth or any other tranforms the API expects.</param>
-        /// <param name="connectorSettings">Connector settings containing cache, numberIsFloat and MaxRows to be returned.</param>        
-        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument, HttpMessageInvoker httpClient, ConnectorSettings connectorSettings)
+        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, ConnectorSettings connectorSettings, OpenApiDocument openApiDocument)
```

Comment:
>AddService [](http://example.com/codeflow?start=50&length=10)

With all the churn ... maybe a clean break and rename the overloads to AddRESTService()? 
Also gives a pattern when we get to AddTabularService...

AddService is a little too generic.

Reply:
I'm not sure yet how we'll implement tabular connectors and would prefer keeping this name of use AddActionConnector instead but that's might not work well with tabular connectors